tests/py/dynamo/conversion/test_matmul_aten.py

```diff
@@ -105,18 +106,18 @@ def forward(self, input):
     @parameterized.expand(
         [
             ("2_2", (2, 3), (3, 2)),
-            # ("2_3", (2, 3), (2, 3, 4)),
-            # ("4_4", (2, 2, 2, 3), (2, 1, 3, 2)),
-            # ("4_2", (2, 1, 2, 3), (3, 2)),
-            # ("2_1", (2, 3), (3,)),
-            # ("1_2", (3,), (3, 2)),
-            # ("1_1", (3,), (3,)),
+            ("2_3", (2, 3), (2, 3, 4)),
+            ("4_4", (2, 2, 2, 3), (2, 1, 3, 2)),
+            ("4_2", (2, 1, 2, 3), (3, 2)),
+            ("2_1", (2, 3), (3,)),
+            ("1_2", (3,), (3, 2)),
+            ("1_1", (3,), (3,)),
         ]
     )
-    def test_matmul(self, _, input_shape, other_shape):
+    def test_matmul_matmul(self, _, input_shape, other_shape):
```

Comment:
Why the renaming here ?

Reply:
this is just to following other testcases naming convention:
test_matmul_mm      ---> torch.ops.aten.mm.default
test_matmul_matmul ---> torch.ops.aten.matmul.default
test_matmul_mv          ---> torch.ops.aten.mv.default